src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -67,6 +67,16 @@
             <argument type="service" id="router" on-invalid="ignore" />
         </service>
 
+        <service id="http_exception_listener" class="Symfony\Component\HttpKernel\EventListener\ExceptionListener">
```

Comment:
Just asking: do we still have to use snake_case ID for our services instead of FQCN id?

```xml
<service id="Symfony\Component\HttpKernel\EventListener\ExceptionListener">
  ...
</service>
```

Reply:
if they don't make sense as autowirable services, yes (to avoid polluting the list of available autowiring types with them).